openmrs/apps/clinical/extension.json

```diff
@@ -174,29 +174,19 @@
     "extensionPointId": "org.bahmni.clinical.conceptSetGroup.observations",
     "type": "config",
     "extensionParams": {
-      "conceptName": "Obstetrics"
+      "formName": "Obstetrics and Gynaecology"
```

Comment:
@SanoferSameera do we have this form configured in our metadata ?

Reply:
Yes. here it is https://github.com/Bahmni/standard-config/blob/master/masterdata/configuration/bahmniforms/obstetrics_and_gynaecology.json